app/components/ak-text-field/index.hbs

```diff
@@ -31,6 +32,7 @@
       data-test-text-input
       id={{this.id}}
       local-class='ak-text-input'
+      class={{@textInputRootClass}}
```

Comment:
this is not required as this element is exposed

Reply:
My styles were not getting applied if was not adding this